src/envoy/http/mixer/control_factory.h

```diff
@@ -82,6 +82,7 @@ class ControlFactory : public Logger::Loggable<Logger::Id::config> {
         case istio::utils::Logger::Level::ERROR_:
           return ENVOY_LOG_CHECK_LEVEL(error);
```

Comment:
```
default:
  NOT_REACHED_GCOVR_EXCL_LINE;
```
is more in line with Envoy's coding style, and it should fix this as well.

Reply:
done